homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: todo
+  config-flow: done
```

Comment:
`            data_schema=vol.Schema({vol.Optional(CONF_ACCESS_TOKEN): str}),` why is the access token optional?